src/tamp_improv/plotting.py

```diff
@@ -0,0 +1,66 @@
+# type: ignore
```

Comment:
can we put this in a new `/scripts` directory instead? i.e., outside of `src/`. and then you can exclude that whole directory from type checking if you want (rather than type: ignore).